File: BackEnd/ProjectManagerSystem/DataModelLibrary/Database/AtomPermissionKind/DbAtomPermissionKindEnum.cs
namespace DataModelLibrary.Database.AtomPermissionKind;

/// <summary>資料庫-原子-權限-類型-列舉</summary>
public enum DbAtomPermissionKindEnum : short
{
    /// <summary>未定義</summary>
    Undefined = 0,

    /// <summary>無權限</summary>
    Denied = 1,

    /// <summary>自身</summary>
    Self = 2,

    /// <summary>所有人</summary>
    Everyone = 3,

}

File: FrontEnd/manager-back-site/src/constants/DbAtomPermissionKindEnum.ts
/** 資料庫-原子-權限-類型-列舉 */
export enum DbAtomPermissionKindEnum {
  /** 未定義 */
  Undefined = 0,
  /** 無權限 */
  Denied = 1,
  /** 自身 */
  Self = 2,
  /** 所有人 */
  Everyone = 3,
}
